test/run-drun/stable-mem-nat8.mo

```diff
@@ -1,7 +1,12 @@
 import P "mo:⛔";
 import StableMemory "stable-mem/StableMemory";
+
+import Region "stable-region/Region";
+
 actor {
 
+  stable var region0 : Region = StableMemory.region(); // TEMP -- prevent GC from collecting our RTS-allocated region0.
```

Comment:
If you have not already extended serialization to deal with Regions this will probably crash. You might be ok just changing the type to `Any` (to skip saving the value, while keeping it live), or removing `stable` altogether.

Reply:
Removed `stable`